Source/CombatExtended/CombatExtended/Projectiles/ProjectileCE.cs

```diff
@@ -1206,9 +1206,16 @@ public override void DrawAt(Vector3 drawLoc, bool flip = false)
             }
             else
             {
+                Quaternion shadowRotation = ExactRotation;
+                Quaternion projectileRotation = DrawRotation;
+                if (def.projectile.spinRate != 0f)
+                {
+                    float num2 = 60f / def.projectile.spinRate;
```

Comment:
This should use the global `GenTicks.TicksPerRealSecond` to convert seconds per rotation to ticks per rotation.

Reply:
Done